README.md

```diff
@@ -1,11 +1,11 @@
 # gRPC Elixir
 
+[![GitHub CI](https://github.com/elixir-grpc/grpc/actions/workflows/ci.yml/badge.svg)](https://github.com/elixir-grpc/grpc/actions/workflows/ci.yml)
 [![Hex.pm](https://img.shields.io/hexpm/v/grpc.svg)](https://hex.pm/packages/grpc)
-[![Travis Status](https://app.travis-ci.com/elixir-grpc/grpc.svg?branch=master)](https://app.travis-ci.com/elixir-grpc/grpc)
```

Comment:
Should we retire the project from Travis CI, as it's still active, see https://app.travis-ci.com/github/elixir-grpc/grpc?

Reply:
Excelent